### PR TITLE
Changed style/design of Notification system.

### PR DESCRIPTION
**Pull request by jainarunarunjain**
@aka-jain Please review this PR.

### CODE REVIEW

File: frontend/src/js/route-config/route-config.js [hunk 1]
```diff
@@ -437,7 +437,7 @@
                 return (typeof arg === undefined ? def : arg);
             }
 
-            timeout = pick(30000000000000000, 3000);
```
Comment (aka-jain) [inline frontend/src/js/route-config/route-config.js]: @jainarunarunjain replace it with... timeout = pick(timeout, 3000);

File: frontend/src/css/modules/notification.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#toast-container {
+	top: 20px;
+	right: 0;
+	> div {
+		opacity: 1;
+		-moz-border-radius: 3px 0px 0px 3px;
+		-webkit-border-radius: 3px 0px 0px 3px;
+		border-radius: 3px 0px 0px 3px;
+		box-shadow: 0 0 6px #000000;
```
Comment (aka-jain) [inline frontend/src/css/modules/notification.scss]: @jainarunarunjain Change box shadow color code to #9d9d9d

File: frontend/src/js/route-config/route-config.js (hunk 1 above)
Comment (jainarunarunjain) [inline frontend/src/js/route-config/route-config.js]: @aka-jain Sure

File: frontend/src/css/modules/notification.scss (hunk 2 above)
Comment (jainarunarunjain) [inline frontend/src/css/modules/notification.scss]: @aka-jain Sure

**Pull request merged**